instance/static/js/src/instance.js

```diff
@@ -23,7 +24,8 @@ var app = angular.module('InstanceApp', [
     'ngRoute',
     'ui.router',
     'restangular',
-    'mm.foundation'
+    'mm.foundation',
+    'ngSanitize'
```

Comment:
@fferrin why is ngSanitize added? I don't see it used anywhere.

Reply:
Oh I see, perhaps that was an alternative to dompurify? Looks like we can strip out ngSanitize if we're going with dompurify?

Reply:
You were right. DOMPurify is an alternative to ngSanitize. I'll remove it